lib/components/myTextField.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:flutter/material.dart';
+
+class MyTextField extends StatelessWidget{
+  
+  final String hintText;
+  final bool filled;
+  final Color filledColor;
+  final Color textColor;
+  final double border;
+  final bool hideText;
+  String value;
+
+  MyTextField(
+    this.hintText,
+    this.filled,
+    this.filledColor,
+    this.textColor,
+    this.border,
+    this.hideText
+  );
+
+@override
+  Widget build(BuildContext context){
+    if(hideText){
```

Comment:
Is this `if` right? You should use a [ternary operator](https://code-maven.com/slides/dart-programming/ternary-operator) for `labelStyle`

Reply:
Nice catch. I didn't know I could do it. I'll update this PR.